### ISSUE TITLE
Mock file simple_mock_model_ext.mof fails on "instance enumerate"

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbemtools#260]
This command fails:
```
$ pywbemcli -m tests/unit/simple_mock_model_ext.mof -o table instance enumerate CIM_Foo   
Error: CIMError: 1 (CIM_ERR_FAILED): Invalid Repository. Multiple instances with same path CIMInstanceName(classname='CIM_Foo_sub', keybindings=NocaseDict({'InstanceID': 'CIM_Foo_sub3'}), namespace='root/cimv2', host=None).
```
When requesting only the names, it succeeds:
```
$ pywbemcli -m tests/unit/simple_mock_model_ext.mof -o table instance enumerate CIM_Foo -o
InstanceNames: CIM_Foo
+--------+-------------+------------------------------------------------+
| host   | namespace   | keybindings                                    |
|--------+-------------+------------------------------------------------|
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo1'})         |
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo2'})         |
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo3'})         |
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo30'})        |
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo31'})        |
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo_sub1'})     |
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo_sub2'})     |
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo_sub3'})     |
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo_sub_sub1'}) |
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo_sub_sub2'}) |
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo_sub_sub3'}) |
+--------+-------------+------------------------------------------------+
```

**Comment by KSchopmeyer** [comment]
Do we have something specific with OS-x since it works on Ubuntu:

```
pywbemcli -m tests/unit/simple_mock_model.mof -o table instance enumerate CIM_Foo
Instances: CIM_Foo
+--------------+---------------+
| InstanceID   | IntegerProp   |
|--------------+---------------|
| "CIM_Foo1"   | 1             |
| "CIM_Foo2"   | 2             |
| "CIM_Foo3"   |               |
+--------------+---------------+
(pywbemtoolstmp) kschopmeyer@leonard:~/pywbem/pywbemtoolstmp/pywbemtools(ks/#257-envvar-and-optionnames)$ pp pywbemcli -m tests/unit/simple_mock_model.mof -o table instance enumerate CIM_Foo -o
InstanceNames: CIM_Foo
+--------+-------------+----------------------------------------+
| host   | namespace   | keybindings                            |
|--------+-------------+----------------------------------------|
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo1'}) |
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo2'}) |
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo3'}) |
+--------+-------------+----------------------------------------+

Sounds like there is something tied to the mock repository that is causing CIM Instance paths to appear to be the same.

**Comment by KSchopmeyer** [comment]
Note that it is something with the model ..._ext where I added a number of instances just a day or so ago to test ordering when we took the sort out.

**Comment by andy-maier** [comment]
We can close this one as pursued in another issue.

**Comment by andy-maier** [comment]
Fixed again in PR #400.
